go/src/socialapi/workers/payment/plan.go

```diff
@@ -21,79 +18,78 @@ const (
 	// planPrefix       = "p_"
 	customPlanPrefix = "p_c_"
 
-	Free        = "p_free"
-	UpTo10Users = "p_up_to_10"
-	UpTo50Users = "p_up_to_50"
-	Over50Users = "p_over_50"
+	Free    = "p_free"
+	Solo    = "p_solo"
+	General = "p_general"
 )
 
-// Plans holds koding provided plans on stripe
-var Plans = map[string]*stripe.PlanParams{
+var planMu sync.RWMutex
```

Comment:
i dont have the stack traces ATM, i saw them while back ago, not while implementing this feature. if you guys want i can remove these and wait for that happen again?